ui/src/app/machines/views/MachineDetails/MachineNetwork/InterfaceFormTable/InterfaceFormTable.tsx

```diff
@@ -119,7 +126,9 @@ const InterfaceFormTable = ({
         {
           content: (
             <>
-              <TableHeader className="u-align--center">PXE</TableHeader>
+              <TableHeader className="u-align--center">
+                {isPrimary ? "Primary" : "PXE"}
```

Comment:
As I understand it `Primary` only makes sense for bonds, since you're determining which of the bond parents' details you want to assign to the new bond.